src/sync.rs

```diff
@@ -44,10 +44,15 @@ pub(crate) struct SyncItems {
 
 impl Context {
     /// Checks if sync messages shall be sent.
-    /// Receiving sync messages is currently always enabled;
-    /// the messages are force-encrypted anyway.
     async fn is_sync_sending_enabled(&self) -> Result<bool> {
-        self.get_config_bool(Config::SendSyncMsgs).await
+        self.get_config_bool(Config::SyncMsgs).await
+    }
+
+    /// Checks if sync messages shall be executed.
+    /// Before this was always enabled, the messages are force-encrypted anyway. But for users it's
+    /// probably easier to understand a "synchronised device" concept than a "master/slave" one.
+    pub(crate) async fn is_sync_exec_enabled(&self) -> Result<bool> {
```

Comment:
Is this a placeholder for later use? Or why do we have to methods with identical body now?

Reply:
well, this PR is mostly a rename and adding an API that is not needed by UIs.  It's also a complete side-track that has no effect on actual user visible functionality.  If we need to refactor it because we decide to expose "firefox sync" style options to users then we can do it when we go for that feature but that is not in the current scope of getting some more chat states synchronized at all.

Reply:
Replaced it with `get_config_bool()` as per @link2xt's suggestion so that it doesn't raise additional questions